internal/otgutils/print.go

```diff
@@ -0,0 +1,70 @@
+package otgutils
```

Comment:
Shouldn't these utilities be used in the tests that are part of the PR?

Reply:
Every test has its own PR and this branch has none (except of topologies/otg_tests/topology_test/topology_test.go) . The tests use otgutils but we decided to track this separately.

Reply:
Every test has its own PR and this branch has none (except of topologies/otg_tests/topology_test/topology_test.go) . The tests use otgutils but we decided to track this separately.